### ISSUE TITLE
"NoSuchMethodError: The getter 'dy' was called on null.\nReceiver: null\nTried calling: dy

### ISSUE THREAD

**Issue by a616781689** [aleksanderwozniak/simple_gesture_detector#1]
![image](https://user-images.githubusercontent.com/31605348/93729558-a27fcd80-fbf7-11ea-99a0-0e7c20282e0a.png)

**Comment by ezamagni** [comment]
Same problem here
<img width="876" alt="Schermata 2020-10-06 alle 13 06 00" src="https://user-images.githubusercontent.com/6697707/95193844-c258e900-07d4-11eb-89d2-064b2d32f569.png">

I suspect this might occur when `SimpleGestureDetector` widget collapses to a size with 0 height or width.

**Comment by aleksanderwozniak** [comment]
Hmm I can't reproduce it myself, even with collapsing size to 0. Do you have some short demo/snippet which you could share?

**Comment by vintage** [comment]
Same issue here. Cannot reproduce it locally, but received report from the Crashlytics about it.

```
Non-fatal Exception: FlutterError
NoSuchMethodError: The getter 'dx' was called on null. Receiver: null Tried calling: dx. Error thrown Instance of 'ErrorDescription'.
_onHorizontalDragEnd

Non-fatal Exception: FlutterError
0  ???                            0x0 _onHorizontalDragEnd + 142 (simple_gesture_detector.dart:142)
1  ???                            0x0 <fn> + 467 (monodrag.dart:467)
2  ???                            0x0 invokeCallback + 183 (recognizer.dart:183)
3  ???                            0x0 _checkEnd + 467 (monodrag.dart:467)
4  ???                            0x0 didStopTrackingLastPointer + 376 (monodrag.dart:376)
5  ???                            0x0 stopTrackingPointer + 342 (recognizer.dart:342)
6  ???                            0x0 _giveUpPointer + 385 (monodrag.dart:385)
7  ???                            0x0 handleEvent + 309 (monodrag.dart:309)
8  ???                            0x0 _dispatch + 77 (pointer_router.dart:77)
9  ???                            0x0 <fn> + 122 (pointer_router.dart:122)
10 ???                            0x0 forEach (dart:collection)
11 ???                            0x0 _dispatchEventToRoutes + 120 (pointer_router.dart:120)
12 ???                            0x0 route + 106 (pointer_router.dart:106)
13 ???                            0x0 handleEvent + 358 (binding.dart:358)
14 ???                            0x0 dispatchEvent + 338 (binding.dart:338)
15 ???                            0x0 dispatchEvent + 267 (binding.dart:267)
16 ???                            0x0 _handlePointerEvent + 295 (binding.dart:295)
17 ???                            0x0 _flushPointerEventQueue + 240 (binding.dart:240)
18 ???                            0x0 _handlePointerDataPacket + 213 (binding.dart:213)
```

**Comment by vintage** [comment]
Any news about the issue @aleksanderwozniak?

**Comment by noinskit** [comment]
+1, I'm also hitting this

**Comment by aleksanderwozniak** [comment]
This has been resolved in version `0.1.5`
